probe/probe.go

```diff
@@ -513,6 +514,12 @@ func readDataFromKubeArmor(c *k8s.Client, o Options, nodeName string) error {
 		log.Println("an error occured when printing output", err)
 		return err
 	}
+	err = setPostureData(buf)
```

Comment:
why are we setting this after printing the output?

Reply:
The output here which is being printed is like Node information, so I thought if there's an error in doing that, there won't be any need to go further and set the posture default values.